pyintegration/deephaven2/replay.py

```diff
@@ -2,12 +2,14 @@
 #   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
 #
 """ This module provides support for replaying historical data. """
+
 import jpy
 
 from deephaven2 import dtypes, DHError
 from deephaven2._wrapper_abc import JObjectWrapper
 from deephaven2.table import Table
-from deephaven2.table_factory import _JReplayer
+
+_JReplayer = jpy.get_type("io.deephaven.engine.table.impl.replay.Replayer")
```

Comment:
Do you still want the `_JReplayer` variable, or do you just want to use `j_object_type`.  I can convince myself either way.

If you remove it, there are other places this pattern is used.

Reply:
I would keep it. Keeping the two names separately makes it a bit more readable and extendable.